packages/web3.js-plugin-fee-estimation/package.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Do we need a dot in the package name?

Reply:
It was a preference for web3.js plugins to have the dot, but I think it'll be fine to remove it

Reply:
unrelated, but ethers.js calles their src folder `src.ts` and it bothers me everytime I see it

Reply:
Apologies, I meant in the package name in the repo, i'm not super opinionated about npm package names, the dot in the directory name is what is throwing me off

Reply:
yea let's keep the package name consistent with web3.js standards but the folder name throws me off as well

Reply:
also remove fee-estimation from the foldername